### ISSUE TITLE
Search box - can't use mouse then Enter key for selections

### ISSUE THREAD

**Issue by brudnak** [rancher/dashboard#8555]
# Setup

| Component                        | Version / Type                                                                     |
| -------------------------------- | ---------------------------------------------------------------------------------- |
| Rancher version                  | v2.7-7913e283ff90eebd1f8cc48860a6b410d72cd4dd-head                                 |
| Rancher commit link              | https://github.com/rancher/rancher/commit/7913e283ff90eebd1f8cc48860a6b410d72cd4dd |
| Installation option              | Helm (high availability)                                                           |
| RKE binary version used          | v1.4.0                                                                             |
| If Helm Chart k8s cluster        | v1.24.6                                                                            |
| Cert Details                     | Let's Encrypt / `nginx`                                                            |
| Docker version                   | 20.10.7, build f0df350                                                             |
| Helm version                     | v2.16.8-rancher2                                                                   |
| Downstream cluster type          | not applicable                                                                     |
| Downstream K8s version           | not applicable                                                                     |
| Authentication providers enabled | local                                                                              |
| Logged in user role              | admin, standard user                                                               |
| Browser type                     | Google Chrome                                                                      |
| Browser version                  | 111.0.5563.110                                                                     |

# Describe the bug

Can't use `Enter` in conjunction with the mouse when selecting namespaces to filter by.

# To Reproduce

1. Start Rancher on 2.7-head
2. Navigate to `Workloads`
3. Click the namespace filter dropdown
4. Using the `mouse` click the `cattle-global-nt` under `Project: System`
5. Now that is selected use the `down arrow` key to navigate to `cattle-system` also under `Project: System`
6. Try pressing `Enter` on `cattle-system`
7. The `Enter` key does not work after 1st using the mouse to make an initial selection

# Result

Can't use `Enter` to select namespaces after using the mouse for an initial selection

# Expected Result

You can use `Enter` to make an initial selection, then use the mouse to make subsequent selections. So I would expect the opposite of that to also work as well.

Enter works for selections if you only use Enter.

See the video below, using a tool to show which keys are being pressed.

# Screenshots 

https://user-images.githubusercontent.com/40877149/228078148-816df1b6-e728-4c64-8937-392abfe4a979.mp4

# Bug Frequency

| Frequency                                                                                          | Selected Option |
| -------------------------------------------------------------------------------------------------- | --------------- |
| Consistent (10/10) - A consistent bug can be replicated every time the test is executed.           | ✅               |
| Frequent (7/10)- A frequent bug can be replicated like 7 out of 10 times when a test is replicated |                 |
| Occasional (5/10) - Occasional bug is more hard to replicate than a frequent bug.                  |                 |
| Once thus far, twice thus far - Few bugs cannot be reproduced every time you run the test.         |                 |

**Comment by gaktive** [comment]
Thanks @brudnak -- based on this not being a blocker, I'm going to push this out to the Q2 release.

**Comment by nwmac** [comment]
Should now be fixed as part of accessibility - please try on latest Rancher and re-open if still an issue.
